integration/client/credentials/credentials_test.go

```diff
@@ -28,7 +28,7 @@ func TestCredentialCreate(t *testing.T) {
 		t.Fatal(err)
 	}
 
-	cred, err := c.CredentialCreate(ctx, reg, "user", "pass")
+	cred, err := c.CredentialCreate(ctx, reg, "user", "pass", false)
```

Comment:
Can you add a test where we test skipping validation?

Reply:
agreed, but let's do it in a follow up pr @jsilverio22. I want to get this one merged for v0.4.1